src/main/java/nextstep/subway/line/domain/Distance.java

```diff
@@ -0,0 +1,57 @@
+package nextstep.subway.line.domain;
+
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+
+@Embeddable
+public class Distance {
+    private static final int MIN_DISTANCE = 0;
+    private static final String CAN_NOT_LESS_THAN_ZERO = "거리는 0이하의 값일 수 없습니다.";
+    @Column(name = "distance", nullable = false)
```

Comment:
```suggestion

    @Column(name = "distance", nullable = false)
```

가독성을 위해 줄바꿈을 제안드립니다 🙏

Reply:
@jhhj424 
다음 단계에서 같이 변경해서 올리도록 하겠습니다!